assets/js/player.js

```diff
@@ -49,6 +49,10 @@ videojs.Vhs.xhr.beforeRequest = function(options) {
 
 var player = videojs('player', options);
 
+player.reloadSourceOnError({
+    errorInterval: 10
+});
```

Comment:
What's that for, exactly? Won't it interfere with `player.on('error', function (event)` (line 147)?

Reply:
Yes it does and in fact I plan to remove the player.error something.

But I'm still trying some things, with `retryOnError` option too. Hence this PR is still "blocked"

Reply:
I've made it so it is only loaded when using dash because it's a dash only feature: https://github.com/videojs/http-streaming/blob/main/docs/reload-source-on-error.md